### PR TITLE
Added trick to save iframe with good quality

### LINKED ISSUE #8273: Unable to open any generative NFT Image
I need to download high-quality Swirls images for the exhibition where we will be exhibiting them. I'm unable to "maximalize" the image. The option is missing:

Screenshot 2023-11-27 at 14 37 05

I'm only able to download 350kb versions.. which is not enough. they will look shitty on big screens.

Screenshot 2023-11-27 at 14 38 21

Please help. The deadline for files for the exhibition is tomorrow.

### PR DESCRIPTION
**Pull request by shashkovdanil**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #8273 
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [ ] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [x] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer?target=16SpvdDgKNiQ3c53DxX7refnQcKUD3uRNim3Z1HBJLNGrtSP&usdamount=0)

#### Community participation

- [x] [Are you at KodaDot Ecosystem Telegram?](https://t.me/kodadot_eco)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at 63cd18f</samp>

Improved iframe scaling in `IFrameMedia` component. Used a wrapper div, a custom hook, and CSS transform to fit iframe content to the container size.

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at 63cd18f</samp>

> _We are the IFrameMedia, we scale the content of the web_
> _We use a wrapper div, we hook the size of the element_
> _We transform the style, we fit the space that we have_
> _We are the IFrameMedia, we are the masters of the embed_

### CODE REVIEW

File: libs/ui/src/components/MediaItem/type/IFrameMedia.vue [hunk 1]
```diff
@@ -1,14 +1,36 @@
...
+      class="iframe-model__wrapper is-flex"
+      :src="animationSrc || src"
+      :alt="alt"
+      sandbox="allow-scripts allow-same-origin allow-modals"
+      allow="accelerometer *; camera *; gyroscope *; microphone *; xr-spatial-tracking *;" />
+  </div>
 </template>
 
 <script lang="ts" setup>
+import { ref } from 'vue'
```
Comment (roiLeo) [inline libs/ui/src/components/MediaItem/type/IFrameMedia.vue:15]:
```suggestion
import { ref, watchEffect } from 'vue'
```

**Review by roiLeo — APPROVED** [review]

**Comment by prury** [comment]
@shashkovdanil i just need to click with the right button and save it, correct?
on canary it saves an image with 950 KB and your deployed version one with 3,96 MB ✔️ 

> btw I fixed this issue as well #8240

yup, it seems fixed

this difference in images in the video below means swirls were somehow zoomed in? 

https://github.com/kodadot/nft-gallery/assets/36627808/cfa360ea-2339-43b3-a408-ffdd03a28c4f

**Comment by roiLeo** [comment]
> ye, and resolution should be 2000x2000

How does it handle `/ahp/collection/40`? as we don't really want end user to load 40 * ~4mb on this page

**Comment by prury** [comment]
> > this difference in images in the video below means swirls were somehow zoomed in?
> 
> Actually it's same pictures but with different quality so you can see more details

let it be noted here then that it will "change" all the nfts 😅

**Review by vikiival — COMMENTED** [review]
Please make sure it works on mobile too

**Review by preschian — APPROVED** [review]

**Comment by vikiival** [comment]
So lets try ❤️

**Comment by vikiival** [comment]
pay 25 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $25 @ 5.34 USD/DOT ~ 4.682 $DOT
🧗 [16SpvdDgKNiQ3c53DxX7refnQcKUD3uRNim3Z1HBJLNGrtSP](https://polkadot.subscan.io/account/16SpvdDgKNiQ3c53DxX7refnQcKUD3uRNim3Z1HBJLNGrtSP)
🔗 [0xbfd484f94a9e451046ce16176d16154b4c2d1dea2fbb0520ad8f8b6777763ee0](https://polkadot.subscan.io/extrinsic/0xbfd484f94a9e451046ce16176d16154b4c2d1dea2fbb0520ad8f8b6777763ee0)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**